zeppelin/zeppelin-cannot-login-domain-joined.md

```diff
@@ -0,0 +1,33 @@
+---
+title: How do I analyze Tez Directed Acyclic Graph (DAG) critical path on HDInsight cluster? | Microsoft Docs
+description: Use the Hive FAQ for answers to common questions on Hive on Azure HDInsight platform.
+keywords: Azure HDInsight, ambari, Tez, FAQ, troubleshooting guide, 
+services: Azure HDInsight
+documentationcenter: na
+author: 
+manager: ''
+editor: ''
+
+
+ms.service: multiple
+ms.workload: na
+ms.tgt_pltfrm: na
+ms.devlang: na
+ms.topic: article
+ms.date: 01/17/2018
+ms.author: NithinM
+---
+
+### TSG Zeppelin Cannot Login 
+#### Issue
+Cannot login to Zeppelin after changing the ADDS password in active directory.
+
+#### Symptoms
+- The user mentioned in the ```activeDirectoryRealm.systemUsername``` of the ```shiro_ini``` file changed the active directory password. 
+- Only the user mentioned in the shiro file is able to login to zeppelin.
+- No other domain joined user can login to zeppelin.
+
+
+#### Resolution Steps
+1.	Verify that the changed password is the root cause by including ```activeDirectoryRealm.systemUsername = <new password>``` in the zeppelin shiro_ini config in ambari. Remove the ```activeDirectoryRealm.hadoopSecurityCredentialPath``` setting.

```

Comment:
typo - activeDirectoryRealm.systemPassword
Also, you can mention how to get to this setting from Ambari UI, may be a snapshot like this - [OneNote](https://microsoft.sharepoint.com/teams/HDInsight96/_layouts/OneNote.aspx?id=%2Fteams%2FHDInsight96%2FSiteAssets%2FHDInsight%20Notebook&wd=target%28Livesite%20Troubleshooting.one%7C8519DDDC-7726-4281-8254-2C19423D8EDA%2FUnable%20to%20login%20to%20Zeppelin%20UI%20after%20AD%20pwd%20changes%7CAF9D0375-DD89-40EB-A7AB-1518B5DB8FE6%2F%29
onenote:https://microsoft.sharepoint.com/teams/HDInsight96/SiteAssets/HDInsight%20Notebook/Livesite%20Troubleshooting.one#Unable%20to%20login%20to%20Zeppelin%20UI%20after%20AD%20pwd%20changes/%20adding%20AD%20users&section-id={8519DDDC-7726-4281-8254-2C19423D8EDA}&page-id={AF9D0375-DD89-40EB-A7AB-1518B5DB8FE6}&end)